### ISSUE TITLE
JuliaHub pricing is intransparent for me

### ISSUE THREAD

**Issue by harald-gegenfurtner** [JuliaComputing/JuliaHub#121]
Hello,

today I have registered for Julia Computing / JuliaHub. Thanks so far.

My problem is that I still don't know your pricing; I don't know *how much exactly* (or even approximately) I'll have to pay per minute for which services. To me, this is the opposite of the "complete transparency" that you advertise.

On your page [](https://juliahub.com/ui/Payments) I can set my price limit and view my balance, but in advance I have not the slightest idea what the actual cost of your service is.

Do I simply miss your price-list, or isn't there any?

Kind regards,

Harald Gegenfurtner
alamak@gmx.de

**Comment by StefanKarpinski** [comment]
You are absolutely right that the pricing is not as clear as we would like it to be at the moment. The job configuration should tell you how much a job will cost but yes, we also need a price list. We're working on that and will publish one soon.

**Comment by harald-gegenfurtner** [comment]
Thank you, Stefan!

**Comment by StatisticalMice** [comment]
Please also make the price list available without needing to create an account like it is with AWS, etc.

**Comment by Dattax** [comment]
We've now got a pricing calculator reflecting costs at the bottom of the pricing page: https://juliahub.com/products/pricing/
